packages/ui/src/2_molecules/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, ReactNode } from 'react';
+
+import styles from './Header.module.css';
+
+type HeaderProps = {
+  dataLayoutId?: string;
+  logo?: ReactNode;
+  menuItems?: ReactNode;
+  secondaryContent?: ReactNode;
+};
+
+export const Header: FC<HeaderProps> = ({
+  dataLayoutId,
+  logo,
+  menuItems,
+  secondaryContent,
+}) => {
+  return (
+    <header data-layout-id={dataLayoutId} className={styles.header}>
+      <div>
+        {logo}
```

Comment:
Logo should be clickable but since it's `ReactNode`, we can just wrap the image with `Link` component or `a` tag. Or would you prefer to have some property for it?

Reply:
nope, will add it